lib/logger.js

```diff
@@ -0,0 +1,66 @@
+const logger = (() => {
+  const checkIfLogsEnabled = () => {
+    const search = window?.location?.search
+    const enabled = search && search.includes('debug=true')
+    global.areLogsEnabled = enabled || false
+    return global.areLogsEnabled
+  }
+
+  const isDev = process.env.NODE_ENV !== 'production'
+
+  const print = (type, ...messages) => {
+    if (typeof global.areLogsEnabled === 'undefined') {
+      checkIfLogsEnabled()
+    }
+
+    if (global.areLogsEnabled || isDev) {
+      switch (type) {
```

Comment:
We will want to enable and disable the different levels of logs. IE if set to "info" then info, warn and error should log while debug and trace are muted

Reply:
Done!